jumpscale/data/serializers/base64.py

```diff
@@ -0,0 +1,20 @@
+import base64
+
+
+def encode(s):
+    if isinstance(s,str):
+        b = s.encode()
+    else:
+        b = s
+    return base64.b64encode(b)
+
+def dumps(s):
```

Comment:
We don't need dumps/loads here, encode/decode is enough.

Reply:
i did it as old code

Reply:
IMHO, old code has bad naming.